test/test_sbom.py

```diff
@@ -0,0 +1,70 @@
+# Copyright (C) 2021 Anthony Harrison
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+import os
```

Comment:
just a nit
```suggestion
from os import path
```
and then we can write `path` instead of `os.path` everywhere